Source/WebCore/display/css/DisplayBoxDecorationPainter.cpp

```diff
@@ -1301,7 +1301,12 @@ void BoxDecorationPainter::paintBoxShadow(PaintingContext& paintingContext, Shad
         auto shadowRectOrigin = fillRect.rect().location() + shadowOffset;
         auto adjustedShadowOffset = shadowRectOrigin - adjustedFillRect.rect().location();
 
-        paintingContext.context.setShadow(adjustedShadowOffset, shadowRadius.value(), resolveColor(shadow.color()), shadow.isWebkitBoxShadow() ? ShadowRadiusMode::Legacy : ShadowRadiusMode::Default);
+        paintingContext.context.setDropShadow({
```

Comment:
Generally we don't do this kind of narrow line wrapping. Please leave them along endless they are overly long.

Reply:
Fixed.